TomestonesItem.yml

```
@@ -7,4 +7,4 @@ fields:
   - name: Tomestones
     type: link
     targets: [Tomestones]
-  - name: Unknown0
+  - name: CurrencyInventorySlot
```

For a field rename, define it as `pendingName`. See https://github.com/xivdev/EXDSchema/blob/main/Usage.md#version-consistency 🙂

```suggestion
  - name: Unknown0
    pendingName: CurrencyInventorySlot
```

---

Thanks Hasel :)